tests/ipfwd/test_nhop_group.py

```diff
@@ -324,6 +324,17 @@ def build_pkt(dest_mac, ip_addr, ttl, flow_count):
     return pkt, exp_packet
 
 
+def validate_asic_route(duthost, route, exist=True):
+    logger.info(f"Checking ip route: {route}")
+    asic_info = duthost.shell(f'redis-cli -n 1 keys "ASIC_STATE:SAI_OBJECT_TYPE_ROUTE_ENTRY:*{route}*"',
```

Comment:
Why do we need `*` here?

Reply:
@bingwang-ms , it's based on the test result.
![image](https://github.com/user-attachments/assets/38145eca-50e4-4ab2-8878-c0a827785199)